test/real_content/swupd_update.bats

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env bats
+
+load "../functional/testlib"
+
+# Test swupd with real official content published on clearlinux.org
+# This test assumes all functional tests to be passing in order to work,
+# because we aren't going to test all swupd functionalities. The idea is
+# to test major swupd commands and standard use cases
+
+#TODO: Add test for crossing a format barrier
+
+# Optional parameters
+# $URL: Clearlinux content and version urls
+# $MAX_VERSIONS: The maximum number of versions we are going to be looking to
+#                test updates. Should be # at least 2 (current + update).
+#                Default is 10.
+# $BUNDLE_LIST: List of bundles to be installed on the system besides os-core
+#               and os-core-update. If empty, install all bundles.
+
+export SWUPD_OPTS # Swupd options to use in tests
+export SWUPD_OPTS_SHORT # Swupd options to use in tests (for bundle-list)
+export FORMAT # Format of the last available version
+export VERSIONS # Array of last available versions. First item is the oldest
```

Comment:
lol, now VERSIONS is back instead of VERSION 😂

Reply:
hahahaha